### PR TITLE
New version: Yemiao.ZhiyunTranslator version 7.7.3

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | 知云文献翻译 |     |
| Version     | 7.7.3     |  |
| Publisher   | 上海叶渺生物科技中心   |       |
| ProductCode | {7F9EF9A6-B29A-49D6-83F1-1720237E7F73}_is1          |     |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59778)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/y/Yemiao/ZhiyunTranslator/7.7.3/Yemiao.ZhiyunTranslator.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+FileExtensions:
+- pdf
+ReleaseDate: 2022-05-08
+Installers:
+- Architecture: x86
+  InstallerUrl: https://cloud.06dn.com/api/v3/file/source/128835/ZhiyunTrans_setup7.7.3A-down.exe
```
Comment (SpecterShell) [inline manifests/y/Yemiao/ZhiyunTranslator/7.7.3/Yemiao.ZhiyunTranslator.installer.yaml:15]:
```suggestion
	InstallerUrl: https://note.youdao.com/yws/api/personal/file/WEB74f25b2dad280e152b81582e4ada67f3?method=download&inline=true&shareKey=cdfe866a8850ddddfe23037089d51ad0
```
From https://free.zhiyunwenxian.cn/zy/UpdateURL.txt

File: manifests/y/Yemiao/ZhiyunTranslator/7.7.3/Yemiao.ZhiyunTranslator.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  InstallerSha256: DD561E9D010E4EB95B6B208822B9A936B2EBDBC828A29006FC0136E250F7EFE3
```
Comment (SpecterShell) [inline manifests/y/Yemiao/ZhiyunTranslator/7.7.3/Yemiao.ZhiyunTranslator.installer.yaml:16]:
```suggestion
  InstallerSha256: AD3B660D0BE2041CEA512C4637CB7744010CDD4B9EC7C5C5F5301376B8FA6F06
```

**Review by ItzLevvie — APPROVED** [review]

**Comment by JohnMcPMS** [comment]
There was a separate PR that got merged already with this update.

**Pull request closed without merging**